verifier/verifier.go

```diff
@@ -176,6 +175,11 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 		if err != nil {
 			return err
 		}
+        pluginVersion := metadata.Version
+        verificationPluginMinVersion, err := getVerificationPluginMinVersion(&outcome.EnvelopeContent.SignerInfo)
+        if versionCompare(verificationPluginMinVersion, pluginVersion) == 1{
```

Comment:
As per https://pkg.go.dev/golang.org/x/mod/semver#Compare should it be 
`versionCompare(pluginVersion, verificationPluginMinVersion, ) >=01` ?

Reply:
I am throwing an error when the verifcationPluginMinVersion is higher than the plugin version in which case the value of versionCompare(verificationPluginMinVersion, pluginVersion) is 1